tools/rostest/src/rostest/rostest_main.py

```diff
@@ -98,6 +98,9 @@ def rostestmain():
     parser.add_option("--results-base-dir", metavar="RESULTS_BASE_DIR",
                       help="The base directory of the test results. The test result file is " +
                            "created in a subfolder name PKG_DIR.")
+    parser.add_option("--port", metavar="PORT",
+                      dest="port", default=None,
```

Comment:
`dest` as well as `default` are not necessary since you choose the default anyway. Please remove them.